js/components/portal-dashboard/questions/multiple-choice-question.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { PureComponent } from "react";
+import { Map } from "immutable";
+import { renderHTML } from "../../../util/render-html";
+
+import css from "../../../../css/portal-dashboard/questions/multiple-choice-question.less";
+
+interface IProps {
+  question?: Map<string, any>;
+}
+
+export default class MultipleChoiceQuestion extends PureComponent<IProps> {
+  render() {
+    const { question } = this.props;
+    const prompt = question?.get("prompt");
+    const choices: Map<string,any> = question?.get("choices") || [];
+
+    return (
+      <div className={css.questionText}>
+      {prompt && renderHTML(prompt)}
+      { choices.size > 0 ? choices.toArray().map(this.renderMultipleChoiceChoices(choices.toArray().length)) : "" }
```

Comment:
Tab in the content inside the div for consistency and readability.

Reply:
Can be rewritten as: 
`{ choices.toArray?.().map(this.renderMultipleChoiceChoices) }`